pygmt/tests/test_clib_loading.py

```diff
@@ -60,3 +63,130 @@ def test_clib_names():
         assert clib_names(freebsd) == ["libgmt.so"]
     with pytest.raises(GMTOSError):
         clib_names("meh")
+
+
+#######################################################################################
+# Tests for clib_full_names
+@pytest.fixture(scope="module", name="os_name")
+def fixture_os_name():
+    """
+    Return the name of the current operating system.
+    """
+    return sys.platform
```

Comment:
```suggestion
```
This pytest fixture is a bit too simple and can probably be removed. Just use `sys.platform` instead for all the tests below.

Reply:
Done in aebc06591708bee4510281a20997db381069a4c8.